src/DacpacTool/Program.cs

```diff
@@ -18,7 +18,7 @@ static async Task<int> Main(string[] args)
                 new Option<string>(new string[] { "--version", "-v" }, "Version of the package"),
                 new Option<FileInfo>(new string[] { "--output", "-o" }, "Filename of the output package"),
                 new Option<SqlServerVersion>(new string[] { "--sqlServerVersion", "-sv" }, () => SqlServerVersion.Sql150, description: "Target version of the model"),
-                new Option<FileInfo[]>(new string[] { "--input", "-i" }, "Input file name(s)"),
+                new Option<string>(new string[] { "--inputfile", "-i" }, "Text file listing all input files"),
```

Comment:
Is there a specific reason you've changed this from a `FileInfo` to a `string`? I believe this can still be a `FileInfo` object since it is still referring to a file on disk right?

Reply:
I feel like there was, but now it doesn't make much sense to me. 😄  I'll change this and confirm it works properly.

Reply:
Seems to be working fine as `FileInfo`, and I agree that makes more sense. There's still a small change -- it's now `Option<FileInfo>` (singular) rather than `Option<FileInfo[]>` (array)